test/org/javarosa/xpath/test/XPathEvalTest.java

```diff
@@ -331,9 +335,20 @@ public void doTests () {
         testEval("round('12345.12345', -2)", 12300.0);
         testEval("round('12350.12345', -2)", 12400.0);
         testEval("round('12345.12345', -3)", 12000.0);
+        // XPath specification tests
         testEval("round('1 div 0', 0)", NaN);
-        // Todo: get round('12345.15', 1) and round('-12345.15', 1) working on Java 8
```

Comment:
AFAIK, this test is contrary to the XPath 3.0 specification, which rounds up, see above.